### ISSUE TITLE
Implementation details of Disjoint CIFAR-10

### ISSUE THREAD

**Issue by phquang** [rahafaljundi/Gradient-based-Sample-Selection#4]
Hi,
I want to clarify some implementation details in the paper versus the configuration provided here for the Disjoint CIFAR-10 benchmark:
1) The backbone used is reduced Resnet18 instead of a full Resnet18
2) The total amount of training data is 50,000 (10,000 per task) instead of total 10k (2k per task)
3) The total amount of data stored is 1,000 (averaged 200 per task)
Thank you
